libexec/scoop-info.ps1

```diff
@@ -60,31 +65,58 @@ if ($manifest.description) {
     $item.Description = $manifest.description
 }
 $item.Version = $version_output
-$item.Website = $manifest.homepage
+if ($bucket) {
+    $item.Bucket = $bucket
+}
+if ($manifest.homepage) {
+    $item.Website = $manifest.homepage.TrimEnd('/')
+}
 # Show license
 if ($manifest.license) {
-    $license = $manifest.license
-    if ($manifest.license.identifier -and $manifest.license.url) {
-        $license = "$($manifest.license.identifier) ($($manifest.license.url))"
+    $item.License = if ($manifest.license.identifier -and $manifest.license.url) {
+        if ($verbose) { "$($manifest.license.identifier) ($($manifest.license.url))" } else { $manifest.license.identifier }
     } elseif ($manifest.license -match '^((ht)|f)tps?://') {
-        $license = "$($manifest.license)"
+        $manifest.license
     } elseif ($manifest.license -match '[|,]') {
-        $licurl = $manifest.license.Split("|,") | ForEach-Object {"https://spdx.org/licenses/$_.html"}
-        $license = "$($manifest.license) ($($licurl -join ', '))"
+        if ($verbose) {
+            "$($manifest.license) ($(($manifest.license -Split "\||," | ForEach-Object { "https://spdx.org/licenses/$_.html" }) -join ', '))"
+        } else {
+            $manifest.license
+        }
     } else {
-        $license = "$($manifest.license) (https://spdx.org/licenses/$($manifest.license).html)"
+        if ($verbose) { "$($manifest.license) (https://spdx.org/licenses/$($manifest.license).html)" } else { $manifest.license }
+    }
+}
+
+if ($manifest.depends) {
+    $item.Dependencies = if ($manifest.depends -is [System.Array]) {
+        $manifest.depends -join ' | '
+    } else {
+        $manifest.depends
+    }
+}
+
+if (Test-Path $manifest_file) {
+    if (Get-Command git -ErrorAction Ignore) {
+        $gitinfo = (git -C (Split-Path $manifest_file) log -1 -s --date=format:'%d-%m-%Y %H:%M:%S' --format='%ad#%an' $manifest_file 2> $null) -Split '#'
+    }
+    if ($gitinfo) {
+        $item.'Updated at' = $gitinfo[0] | Get-Date
```

Comment:
```suggestion
        $item.'Updated at' = $gitinfo[0] | Get-Date -Format 'yyyy-MM-dd HH:mm:ss'
```

Reply:
Doing this will change the field's type from `System.DateTime` to `System.String`.